README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Shawn's Brief Biography:
+- **Major**: *Philosophy* & *Political Science*
+- **Year**: *Senior*
+- **Currently located**: *New Jersey*
+- Have you ever wondered if the Divorce rate in Maine correlates with the Per capita consumption of margarine? Fear not, check out this website on: [spurious relationships](https://www.tylervigen.com/spurious-correlations)
```

Comment:
Awesome link. And nice recognition of correlation. We'll be talking about that a little bit later on in the course :).